Makefile

```diff
@@ -237,9 +237,8 @@ verify: .init .generate_files verify-client-gen
 	@bash -c '[ "`cat .out`" == "" ] || (cat .out ; false)'
 	@rm .out
 	@#
-	#disabled because of so many flakes during PR verifications
-	#@echo Running href checker:
-	#@$(DOCKER_CMD) verify-links.sh .
+	@echo Running href checker:
+	@$(DOCKER_CMD) verify-links.sh -t .
```

Comment:
For other reviewers: new `-t` argument:
```
-t   retry GETs to http(s) URLs 5 times
```
https://github.com/duglin/vlinker/blob/master/bin/verify-links.sh#L58

Reply:
Does `-t` take an argument? Maybe a feature request.

Do we want either of the `-v` or `-d` flags in addition?

Reply:
I thought about making -t take an arg, but decided to save that for later if we think we need it.  -v might be handy if you think people want to see each file listed as its scanned, but it could be kind of verbose.  I tend to prefer quiet things until something fails, then I add flags to debug.